### ISSUE TITLE
Deleted and edited messages

### ISSUE THREAD

**Issue by bluefiberbread** [Tyrrrz/DiscordChatExporter#48]
If somebody deletes or edits their message, will I be able to see the original in the logs after updating the logs?

**Comment by Tyrrrz** [comment]
What do you mean by updating the logs?

**Comment by bluefiberbread** [comment]
Oh I thought if I exported a log from a channel, and somebody deleted/edited their message, and I exported it again, it'd show up as edited/deleted with the original message intact. I realized that's not the case now

**Comment by Tyrrrz** [comment]
No, it can't do that. But if you save logs separately each time, in plaintext, you would be able to use diff tools (e.g. WinMerge) to compare them and see if anything was edited or deleted since the last export.
